src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -314,7 +322,7 @@
     height: 100%;
   }
 
-  &.pf-m-split-button {
+  @mixin menuToggleDiv {
```

Comment:
I don't know that you need/want all these styles to apply to the typeahead - it's probably just the first 2 lines

```
    --pf-c-menu-toggle__toggle-icon--MarginRight: 0;

    padding: 0; // pass padding to children
```

The other styles for the children I think are too specific to the split button to be considered shared and to share the same vars - keeping in mind it may be breaking to give the typeahead its own vars later as either the typeahead or split button change.

Also instead of using a mixin, I'd group the selectors you want to have these common styles - that's DRYer than this approach since sass will actually spit out this grouping of styles twice.